cmd/chainsimulator/main.go

```diff
@@ -142,6 +147,8 @@ func startChainSimulator(ctx *cli.Context) error {
 		Value:    uint64(cfg.Config.Simulator.RoundsPerEpoch),
 	}
 
+	numOfShards := uint32(cfg.Config.Simulator.NumOfShards)
```

Comment:
Instead of changing the config, you could re-assign the value of `numOfShards` to 1 in case of sovereign

Reply:
I re-assign the value of `numOfShards` below